byron/ledger/executable-spec/src/Ledger/Core.hs

```diff
@@ -31,8 +33,9 @@ import Data.AbstractSize
 -- | An encoded hash of part of the system.
 newtype Hash = Hash
   { unHash :: Int
-  } deriving stock (Show)
-    deriving newtype (Eq, Ord, Hashable, HasTypeReps)
+  } deriving stock (Show, Generic)
```

Comment:
Great! Maybe we could add a small comment here and in the `HasTypeReps` class saying why we should derive using the `anyclass` strategy.

Reply:
👍